From 92f5a4b113deca14e80c218e6bcd06835c3e059f Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 28 Feb 2015 17:17:52 -0500
Subject: [PATCH] tests: fix header inclusion

gdnametest uses string funcs, so pull in string.h.  It also can't pull
in test_config.h directly as it breaks when building out of tree.  Use
the -I paths to find it.

URL: https://bugs.gentoo.org/540376

diff --git a/tests/gdimagefile/gdnametest.c b/tests/gdimagefile/gdnametest.c
index dd8f019..f2bb8a6 100644
--- a/tests/gdimagefile/gdnametest.c
+++ b/tests/gdimagefile/gdnametest.c
@@ -1,9 +1,10 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #include "gd.h"
 #include "gdtest.h"
-#include "../test_config.h"
+#include "test_config.h"
 
 #define WIDTH 60
 #define HEIGHT 50
-- 
2.3.1

